packages/schematics/angular/utility/json-utils.ts

```diff
@@ -0,0 +1,114 @@
+/**
```

Comment:
Why is still file green and not moved? Migrations is also in the same package; no use in keeping two files with the same code in the same package...

Reply:
addressed